experiments/integrated/ozark/ozark_domain.jl

```diff
@@ -1,10 +1,16 @@
 # Domain setup
 # For soil column
-nelements = 60
+nelements = 10
 zmin = FT(-2)
 zmax = FT(0)
-land_domain =
-    LSMSingleColumnDomain(; zlim = (zmin, zmax), nelements = nelements)
+dz_bottom = FT(0.5)
+dz_top = FT(0.025)
```

Comment:
Question / something to consider: 
We define dz top and bottom here, and if I understand correctly the other dz in between are exponential. 
Would it be helpful to add and option for other extrapolation, e.g., linear from top to bottom dz? (just to keep in mind, but no need to do this in this PR)

Reply:
yes! ok, i had the same question. right now there are 3 grid stretching options in ClimaCore: Uniform, Exponential, and GeneralizedExponential. The way I wrote this PR currently makes it so, as you wrote, we have to choose the GeneralizedExponential. From a user interface point of view I thought it was a little cleaner (the user passes `dz_tuple = (dz_bottom, dz_top)`, instead of `stretch = ClimaCore.Meshes.GeneralizedExponentialStretching(dz_bottom, dz_top)`. But, if they pass the stretch object itself, then we have more flexibility on what they can choose (including, in the future, maybe a linear stretching of the grid). 


and then, actually, we do not need an if statement, because we can make the default value `stretch = ClimaCore.Meshes.Uniform()` and always pass `stretching` into the function that makes the mesh.
`mesh = ClimaCore.Meshes.IntervalMesh(column; nelems = nelements, stretch = stretch)`

Im open to the latter! what do you think?

Reply:
I am also ok with both options... 
I guess the question is, do we need these options
For now I am ok to just keep it simple, as it is now.

@braghiere

Reply:
ok, let's leave it for now! Right now exponential stretching is the only option, and if we need other options, we'd need to add them in ClimaCore, and that would most likely be a low priority item anyways.